src/app/shared/comcol/comcol-page-browse-by/comcol-page-browse-by.component.html

```diff
@@ -2,9 +2,11 @@ <h2 class="comcol-browse-label h5">{{'browse.comcol.head' | translate}}</h2>
 <nav class="comcol-browse mb-4" aria-label="Browse Community or Collection">
   <div class="d-none d-sm-block">
 
-    <div class="list-group list-group-horizontal">
+    <div class="list-group list-group-horizontal" role="tablist">
       <a *ngFor="let option of allOptions"
+         [attr.aria-current]="(currentOptionId$ | async) === option.id"
```

Comment:
Strangely, this doesn't entirely work on the `dspace-7_x` branch.  

When the tab is NOT enabled, I *do* see `aria-current=false`.  
But, when a tab *is enabled*, the `aria-current` attribute **disappears**. On the main branch, I see `aria-current=true`.